docs/build/pulse.ipynb

```diff
@@ -779,7 +838,7 @@
     "\n",
     "The `acquire` instruction triggers data acquisition for readout. It takes a duration, an `AcquireChannel` which maps to the qubit being measured, and a `MemorySlot` or a `RegisterSlot`. The `MemorySlot` is classical memory where the readout result will be stored. The `RegisterSlot` maps to a register in the control electronics which stores the readout result for fast feedback.\n",
     "\n",
-    "The `acquire` instructions can also take custom `Discriminator`s and `Kernel`s as keyword arguments."
+    "The `acquire` instruction can also take custom `Discriminator`s and `Kernel`s as keyword arguments. If you are interested in changing the measurement pulse envelope, the `Kernel` subroutine integrates time series of measurement response and generates an IQ data point, which will be classified into a quantum state by the discriminator. This indicates that if a user uses custom measurement stimulus, as in a measurement pulse, one may need to update the kernel setting not to deteriorate the measurement SNR."
```

Comment:
```suggestion
    "The `acquire` instruction can also take custom `Discriminator`s and `Kernel`s as keyword arguments. The `Kernel` subroutine integrates time series of measurement response and generates an IQ data point, which will be classified into a quantum state by the discriminator. This indicates that if a user uses custom measurement stimulus, as in a measurement pulse, one may need to update the kernel setting not to deteriorate the measurement SNR."
```

Reply:
Fixed in https://github.com/Qiskit/documentation/pull/217/commits/098faaefb0af3a0f60ffb232ee70dd8371b0c708